mix.exs

```diff
@@ -13,17 +20,28 @@ defmodule NervesSystemRpi0.Mixfile do
      description: description(),
      package: package(),
      deps: deps(),
-     aliases: ["deps.precompile": ["nerves.env", "deps.precompile"]]]
+     aliases: ["deps.precompile": ["nerves.env", "nerves.precompile", "deps.precompile"],
```

Comment:
Oops. This is the normal `mix` hack to make things work. I think that I missed when this line changed into two additional items for everything.